AssertionError line 239 in Utils.py

<!-- DO NOT DELETE THIS! 
This template is used to facilitate issue resolution.
All text in <!-> tags will not be displayed.
-->

### Summary
<!-- If you are having conversion troubles, please share as much
relevant information as possible. This includes, but is not limited to:
- log of conversion
- heuristic
--> 
```
Traceback (most recent call last):
  File "/home/ggilmore/.local/bin/heudiconv", line 11, in <module>
    load_entry_point('heudiconv==0.5.4.dev1', 'console_scripts', 'heudiconv')()
  File "/home/ggilmore/.local/lib/python2.7/site-packages/heudiconv/cli/run.py", line 127, in main
    process_args(args)
  File "/home/ggilmore/.local/lib/python2.7/site-packages/heudiconv/cli/run.py", line 341, in process_args
    dcmconfig=args.dcmconfig,)
  File "/home/ggilmore/.local/lib/python2.7/site-packages/heudiconv/convert.py", line 199, in prep_conversion
    dcmconfig=dcmconfig,)
  File "/home/ggilmore/.local/lib/python2.7/site-packages/heudiconv/convert.py", line 318, in convert
    treat_infofile(scaninfo)
  File "/home/ggilmore/.local/lib/python2.7/site-packages/heudiconv/utils.py", line 252, in treat_infofile
    j_pretty = json_dumps_pretty(j_slim, indent=2, sort_keys=True)
  File "/home/ggilmore/.local/lib/python2.7/site-packages/heudiconv/utils.py", line 239, in json_dumps_pretty
    "Values differed when they should have not. "\
AssertionError: Values differed when they should have not. Report to the heudiconv developers
```

This works for all my scan sequences except I get this assertion failure on my CT study series. Could this be an issue with the CT sequence?

### Platform details:

Choose one:
- [x ] Local environment
<!-- If selected, please provide OS and python version -->
Ubuntu 16.04, python 2.7
- [ ] Container
<!-- If selected, please provide container name and tag"-->

- Heudiconv version: 
<!-- To check: run heudiconv with just the --version flag -->
version 0.5.4.dev1

--- a/heudiconv/utils.py
+++ b/heudiconv/utils.py
@@ -251,7 +251,17 @@ def treat_infofile(filename):
         j = json.load(f)
 
     j_slim = slim_down_info(j)
-    j_pretty = json_dumps_pretty(j_slim, indent=2, sort_keys=True)
+    dumps_kw = dict(indent=2, sort_keys=True)
+    try:
+        j_pretty = json_dumps_pretty(j_slim, **dumps_kw)
+    except AssertionError as exc:
+        lgr.warning(
+            "Prettyfication of .json failed (%s).  "
+            "Original .json will be kept as is.  Please share (if you could) "
+            "that file (%s) with HeuDiConv developers"
+            % (str(exc), filename)
+        )
+        j_pretty = json.dumps(j_slim, **dumps_kw)
 
     set_readonly(filename, False)
     with open(filename, 'wt') as fp:

